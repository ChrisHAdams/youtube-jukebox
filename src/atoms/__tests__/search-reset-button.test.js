import React from "react"
import renderer from "react-test-renderer"

import SearchResetButton from "../search-reset-button";

describe("SearchResetButton", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SearchResetButton/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})