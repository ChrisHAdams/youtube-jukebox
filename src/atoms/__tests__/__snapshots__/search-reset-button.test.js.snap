// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResetButton renders correctly 1`] = `
<button
  className="sc-bdVaJa gQnvFe"
  data-testid="searchResetButton"
>
  Reset
</button>
`;
